akka-stream/src/main/scala/akka/stream/impl/ActorFlowMaterializerImpl.scala

```diff
@@ -74,7 +74,7 @@ private[akka] case class ActorFlowMaterializerImpl(override val settings: ActorF
             mat
 
           case stage: StageModule ⇒
-            val (processor, mat) = processorFor(stage, effectiveAttributes, calcSettings(effectiveAttributes)(settings))
+            val (processor, mat) = processorFor(stage, effectiveAttributes.and(OperationAttributes.materializationName(flowName)), calcSettings(effectiveAttributes)(settings))
```

Comment:
I'll try to figure out another way to get this info to the Stage, but not sure how much I can without changing too much in the materializer hm.